apps/omg_watcher/test/integration/inflight_exit_test.exs

```diff
@@ -71,5 +73,46 @@ defmodule OMG.Watcher.Integration.WatcherApiTest do
 
     assert {:ok, [%{initiator: ^alice_address, txhash: ^in_flight_tx_hash}]} =
              OMG.Eth.RootChain.get_in_flight_exits(0, eth_height)
+
+    Eth.DevHelpers.wait_for_root_chain_block(eth_height + 10)
+
+    tx_double_spend = API.TestHelper.create_encoded([{blknum, txindex, 0, alice}], @eth, [{alice, 2}, {alice, 3}])
+    assert {:error, {:client_error, %{"code" => "submit:utxo_not_found"}}} = Client.submit(tx_double_spend)
+
+    deposit_blknum = DepositHelper.deposit_to_child_chain(alice.addr, 10)
+    tx = API.TestHelper.create_encoded([{deposit_blknum, 0, 0, alice}], @eth, [{alice, 7}, {alice, 3}])
+    {:ok, %{blknum: tx_blknum, tx_hash: _tx_hash}} = Client.submit(tx)
+
+    in_flight_exit_info =
+      tx
+      |> Base.encode16(case: :upper)
+      |> TestHelper.get_in_flight_exit()
+
+    {:ok, %{"status" => "0x1", "blockNumber" => eth_height}} =
+      Eth.RootChain.in_flight_exit(
+        in_flight_exit_info["in_flight_tx"],
+        in_flight_exit_info["input_txs"],
+        in_flight_exit_info["input_txs_inclusion_proofs"],
+        in_flight_exit_info["in_flight_tx_sigs"],
+        alice.addr
+      )
+      |> Eth.DevHelpers.transact_sync!()
+
+    Eth.DevHelpers.wait_for_root_chain_block(eth_height + 10)
```

Comment:
still this `+ 10` here, should be parametrized, see https://github.com/omisego/elixir-omg/pull/352#discussion_r247893668

also the other comment seems unhandled: https://github.com/omisego/elixir-omg/pull/352#discussion_r247894084